drivers/gl_context/glad/glad.h

```diff
@@ -150,12 +150,22 @@ typedef unsigned int GLhandleARB;
 typedef unsigned short GLhalfARB;
 typedef unsigned short GLhalf;
 typedef GLint GLfixed;
+#if defined(__APPLE__)
```

Comment:
This is automatically generated code. If it does not work on OSX, that should be discussed and fixed upstream (https://github.com/Dav1dde/glad), not here, as it would make maintenance harder.